features/gezag-minderjarige/bevoegdheid-tot-gezag.feature

```diff
@@ -0,0 +1,112 @@
+  Een persoon is onbevoegd tot gezag wanneer die:
+  - is overleden
+  - onder curatele staat
+  - is minderjarig
+
+
+  Achtergrond:
+    Gegeven de persoon 'Gerda' met burgerservicenummer '000000012'
+    * is meerderjarig
+    En de persoon 'Aart' met burgerservicenummer '000000024'
+    * is meerderjarig
+    En de persoon 'Ariana' met burgerservicenummer '000000036'
+    * is meerderjarig
+    En de persoon 'Bert' met burgerservicenummer '000000048'
+    * is minderjarig
+
+
+  Regel: Er is sprake van eenhoofdig ouderlijk gezag als één van de ouders met tweehoofdig ouderlijk gezag overleden of niet bevoegd is
+  
+    Scenario: Minderjarige heeft twee ouders en één van de ouders staat onder curatele
+      Gegeven 'Gerda' en 'Aart' zijn getrouwd
+      En persoon 'Bert'
+      * heeft 'Gerda' als ouder
+      * heeft 'Aart' als ouder
+      En 'Gerda' staat onder curatele
+      Als gezag wordt gevraagd met het burgerservicenummer van 'Bert'
+      Dan is het gezag over 'Bert' eenhoofdig ouderlijk gezag met ouder 'Aart'
+
+
+  Regel: Er is tijdelijk geen gezag als beide ouders met tweehoofdig ouderlijk gezag overleden of niet bevoegd zijn
+  
+    Abstract Scenario: Minderjarige heeft twee ouders en beide ouders staan onder curatele
+      Gegeven 'Gerda' en 'Aart' zijn getrouwd
+      En persoon 'Bert'
+      * heeft 'Gerda' als ouder
+      * heeft 'Aart' als ouder
+      En 'Gerda' <bevoegdheid Gerda>
+      En 'Aart' <bevoegdheid Aart>
+      Als gezag wordt gevraagd met het burgerservicenummer van 'Bert'
+      Dan is er tijdelijk geen gezag over 'Bert' met de toelichting '<toelichting>'
+
+      Voorbeelden:
+      | bevoegdheid Gerda    | bevoegdheid Aart     | toelichting                                                                                    |
+      | is overleden         | is overleden         | Tijdelijk geen gezag omdat beide ouders zijn overleden                                         |
+      | staat onder curatele | staat onder curatele | Tijdelijk geen gezag omdat beide ouders onder curate staan                                     |
+      | is minderjarig       | is minderjarig       | Tijdelijk geen gezag omdat beide ouders minderjarig zijn                                       |
+      | is overleden         | staat onder curatele | Tijdelijk geen gezag omdat een ouder overleden is en de andere ouder onder curate staat        |
+      | is overleden         | is minderjarig       | Tijdelijk geen gezag omdat een ouder overleden is en de andere ouder minderjarig is            |
+      | staat onder curatele | is minderjarig       | Tijdelijk geen gezag omdat een ouder onder curatele staat is en de andere ouder minderjarig is |
+  
+
+  Regel: Er is tijdelijk geen gezag als de ouder met eenhoofdig ouderlijk gezag overleden of niet bevoegd is
+  
+    Scenario: Minderjarige is erkend voor 1 januari 2023 en de geboortemoeder is overleden
```

Comment:
Is '1 januari 2023' een datum dat een regel/wet is ingegaan? Hoe ziet gezag voor deze minderjarige eruit als hij ná 1 januari 2023 is erkend?

Reply:
.....en te verwijzen naar het scenario op regel 26 om duidelijk te maken wat er gebeurt bij erkenning na 1 jan 2023.

Reply:
we hebben het scenario vereenvoudigd naar een éénouder situatie. dan is verdere uitleg of verwijzing niet nodig